core/touch_gesture.js

```diff
@@ -55,10 +55,10 @@ Blockly.TouchGesture = function(e, creatorWorkspace) {
 
   /**
    * A map of cached points used for tracking multi-touch gestures.
-   * @type {Object<number|string, Blockly.utils.Coordinate>}
+   * @type {!Object<number|string, Blockly.utils.Coordinate>}
    * @private
    */
-  this.cachedPoints_ = {};
+  this.cachedPoints_ = Object.create(null);
```

Comment:
I'm curious why use Object.create() instead of `{}`?

Reply:
Be more explicit that this is used purely as a map, ie: not an object with a prototype.